.github/workflows/run-entry-points.yml

```
@@ -27,3 +27,5 @@ jobs:
       run: |
         yaqd-omega-platinum --version
         yaqd-omega-platinum --help
+	yaqd-omega-iseries-modbus --version
+	yaqd-omega-iseries-modbus --help
```

```suggestion
        yaqd-omega-iseries-modbus --version
        yaqd-omega-iseries-modbus --help
```

Tabs to spaces for consistency in the yml file